lib/logstash/filters/mutate.rb

```diff
@@ -268,7 +273,11 @@ def convert(event)
       else
         value = converter.call(original)
       end
-      event[field] = value
+      if value.nil?
```

Comment:
I guess not, but then kinda out of scope for this PR since it might break current behavior. Anyway, since it did not break any other spec, we can merge this